.gitignore

```diff
@@ -0,0 +1,3 @@
+/go.mod
```

Comment:
I think you probably don't want to ignore these... if you update a go module dependency, then go will generate a line in here.  Then in `go.sum` it record the digest of what it downloaded to build with.  I think.

Reply:
ok